GiEnJul/Controllers/VerifyConnectionController.cs

```diff
@@ -47,11 +47,6 @@ public async Task VerifyConnection(string giverRowKey, string recipientRowKey, s
             recipient.FamilyMembers = await _personRepository.GetAllByRecipientId(recipient.RowKey);
             var giver = await _giverRepository.GetGiverAsync(partitionkey, giverRowKey);
 
-            if (await _connectionRepository.ConnectionExists(giver, recipient))
-            {
```

Comment:
Keep this if, as its not necessary to continue if the connection is already made.

Put a return inside the if